arm/Microsoft.Insights/privateLinkScopes/readme.md

```diff
@@ -26,6 +26,42 @@ This module deploys Azure Monitor Private Link Scope
 | `scopedResources` | array | `[]` |  | Optional. Configuration Details for Azure Monitor Resources. |
 | `tags` | object | `{object}` |  | Optional. Resource tags. |
 
+### Parameter Usage: `privateEndpoints`
```

Comment:
Double up. Remove one of the privateEndpoints

Reply:
done